styleguide.config.js

```diff
@@ -1,130 +1,191 @@
 const path = require('path')
-const pkg = require('./package.json');
-const homePageComponents = ['src/lib/containers/home_page/**/[A-Z]*.tsx', 'src/lib/containers/widgets/themes-plot/**/ThemesPlot.tsx', 'src/lib/containers/**/UserCardListRotate.tsx', 'src/lib/containers/**/RssFeedCards.tsx', 'src/lib/containers/**/TableFeedCards.tsx', 'src/lib/containers/**/UpsetPlot.tsx', 'src/lib/containers/**/MarkdownSynapse.tsx']
-const ignoreComponents = ['**/*.Mobile.tsx','**/*.Desktop.tsx']
+const {
+  createWebpackDevConfig,
+  createWebpackProdConfig,
+} = require('@craco/craco')
+const { NormalModuleReplacementPlugin } = require('webpack')
+
+const pkg = require('./package.json')
+const cracoConfig = require('./craco.config.js')
+const webpackConfig =
```

Comment:
We have to pass the Webpack config created by craco to React-Styleguidist for it all to work